Kiosk/App/CardHandler.swift

```diff
@@ -1,9 +1,14 @@
 import UIKit
-import ReactiveCocoa
+import RxSwift
 
 class CardHandler: NSObject, CFTReaderDelegate {
 
-    let cardSwipedSignal = RACSubject()
+    private let _cardStatus = BehaviorSubject(value: "")
```

Comment:
Not sure I understand the pattern here -- why does this variable have an underscore, and why does the Observable below close over it?